spyql/qdict.py

```diff
@@ -0,0 +1,91 @@
+from spyql.nulltype import NULL
+import spyql.log
+
+
+class qdict(dict):
+    """
+    A dictionary that supports :data:`~spyql.nulltype.NULL` where items can
```

Comment:
❓ What does the `~` mean?

Reply:
does not print the full path, only prints `NULL`